CHANGELOG.md

```diff
@@ -1,4 +1,22 @@
-# dbt_linkedin_source version.version
+# dbt_linkedin_source v0.11.0
+
+[PR #74](https://github.com/fivetran/dbt_linkedin_source/pull/74) includes the following updates:
+
+## Schema Changes
+**6 total changes • 0 possible breaking changes
+| Data Model                                      | Change Type | Old Name | New Name                                  | Notes                                                             |
+|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| stg_tiktok_ads__geo_tmp       | New temp model   |          |  | Temp model added for `geo`.               |
```

Comment:
Similar comment to add the links to the DAG.

Reply:
Updated